examples/apps/attributable-map/src/view.ts

```diff
@@ -0,0 +1,141 @@
+	countersContainerDiv.style.marginBottom = "2rem";
+	wrapperDiv.appendChild(countersContainerDiv);
+
+	const redCounterContainerDiv = document.createElement("div");
+	redCounterContainerDiv.style.fontSize = "5rem";
+	redCounterContainerDiv.style.fontWeight = "bold";
+	redCounterContainerDiv.style.textAlign = "center";
+	countersContainerDiv.appendChild(redCounterContainerDiv);
+
+	const greenCounterContainerDiv = document.createElement("div");
+	greenCounterContainerDiv.style.fontSize = "5rem";
+	greenCounterContainerDiv.style.fontWeight = "bold";
+	greenCounterContainerDiv.style.textAlign = "center";
+	countersContainerDiv.appendChild(greenCounterContainerDiv);
+
+	// Create container div for buttons and add styling
+	const buttonsContainerDiv = document.createElement("div");
+	buttonsContainerDiv.style.display = "flex";
+	buttonsContainerDiv.style.width = "100%";
+	buttonsContainerDiv.style.justifyContent = "space-around";
+	buttonsContainerDiv.style.marginBottom = "2rem";
+	wrapperDiv.appendChild(buttonsContainerDiv);
+
+	// Create red button element and add styling
+	const redButton = document.createElement("button");
+	redButton.style.fontSize = "2rem";
+	redButton.style.padding = "1rem 2rem";
+	redButton.style.borderRadius = "1rem";
+	redButton.style.backgroundColor = "#F84F31";
+	redButton.style.color = "white";
+	redButton.style.border = "none";
+	redButton.style.cursor = "pointer";
+	redButton.textContent = "Hit";
+	buttonsContainerDiv.appendChild(redButton);
+
+	redButton.addEventListener("click", () => {
+		hitCounter.hit("red");
+	});
+
+	// Create green button element and add styling
+	const greenButton = document.createElement("button");
+	greenButton.style.fontSize = "2rem";
+	greenButton.style.padding = "1rem 2rem";
+	greenButton.style.borderRadius = "1rem";
+	greenButton.style.backgroundColor = "#23C552";
+	greenButton.style.color = "white";
+	greenButton.style.border = "none";
+	greenButton.style.cursor = "pointer";
+	greenButton.textContent = "Hit";
+	buttonsContainerDiv.appendChild(greenButton);
+
+	greenButton.addEventListener("click", () => {
+		hitCounter.hit("green");
+	});
+
+	// Create container div for attribution text and add styling
+	const attributionContainerDiv = document.createElement("div");
+	attributionContainerDiv.style.display = "flex";
+	attributionContainerDiv.style.width = "100%";
+	attributionContainerDiv.style.justifyContent = "space-around";
+	attributionContainerDiv.style.marginBottom = "2rem";
+	wrapperDiv.appendChild(attributionContainerDiv);
+
+	const redAttributionContainerDiv = document.createElement("div");
+	redAttributionContainerDiv.style.fontSize = "2rem";
+	redAttributionContainerDiv.style.fontStyle = "italic";
+	redAttributionContainerDiv.style.textAlign = "center";
+	redAttributionContainerDiv.style.marginBottom = "2rem";
+	attributionContainerDiv.appendChild(redAttributionContainerDiv);
+
+	const greenAttributionContainerDiv = document.createElement("div");
```

Comment:
one thing it would be nice to include is an element which shows the name of the current user on each tab, which makes subsequent attribution properties more clear.

Reply:
don't have to fix this now, but IMO worth filing some work for. It's a nice QoL improvement.